addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -3085,6 +3085,8 @@ options.registry.ConditionalVisibility = options.Class.extend({
     async onTargetShow() {
         this.$target[0].classList.remove('o_conditional_hidden');
     },
+    // Todo: remove me in master as this is now handled at the "Snippet Editor"
+    // level.
```

Comment:
We cannot remove function in stable you are right :+1: . But here can we already remove `this.trigger_up('snippet_option_visibility_update', { show: true });` as it is useless ? Or I miss something ?